### DISCUSSION TITLE
Unable to close a connection

### DISCUSSION THREAD

**Discussion by rshunim** [sqlalchemy/sqlalchemy#11283] [Usage Questions]
### Describe the bug

when using the method .close() in my code (on **sqlalchemy.engine.base.Connection** object), and then checking the connections in the container, I still see the connection.
<img width="641" alt="image" src="https://github.com/sqlalchemy/sqlalchemy/assets/102469772/ecd2d3e7-0e8d-4a21-b3a3-092198720d0b">
Is there a problem with this method?

### SQLAlchemy Version in Use

20.0.29

### DBAPI (i.e. the database driver)

pyodbc

### Database Vendor and Major Version

Teradata

### Python Version

3.10

### Operating system

Linux

### To Reproduce

```python
# sqlalchemy.engine.base.Connection - Object
client.connection.close()
# after closing still see the connection in the container
```

### Error

There is no error

**Comment by zzzeek** [comment]
SQLAlchemy's engine [uses a connection pool by default](https://docs.sqlalchemy.org/en/20/core/engines.html).  To dispose of this connection pool, use [engine.dispose()](https://docs.sqlalchemy.org/en/20/core/connections.html#engine-disposal).  to turn off the pooling so that connections are closed immediately, [use NullPool](https://docs.sqlalchemy.org/en/20/core/pooling.html#switching-pool-implementations)

**Comment by rshunim** [comment]
I want to use connection pool with specific ttl for connection. is that possible?

**Reply by zzzeek** [reply]
we call TTL "pool_recycle" so that is [here](https://docs.sqlalchemy.org/en/20/core/pooling.html#setting-pool-recycle)
